ui/src/button.tsx

```diff
@@ -125,6 +135,20 @@ const
           wave.push()
         }
       },
+      onClick = handleOnClick(name, value, path),
+      getItemProps: (commands: Command[]) => Fluent.IContextualMenuItem[] = commands =>
```

Comment:
Wondering whether we cannot reuse the existing [toCommands](https://github.com/h2oai/wave/blob/ced07d4a630d898da5f544e283e4d7890f0b3abe/ui/src/toolbar.tsx#L87) function that solves this already. It should already have styling (instead of your own `subComponentStyles`) and onClick figured out.

Reply:
Nice to find that almost identical function as I've implemented already exists. It's reused and it works. Thanks!